composer.json

```diff
@@ -17,20 +17,22 @@
         }
     ],
     "require": {
-        "php": "^7.2",
+        "php": "^7.3 | ^8.0",
```

Comment:
why increasing php from 7.2 to 7.3?

Reply:
yep, phpunit 9 requirers >=7.3 https://packagist.org/packages/phpunit/phpunit

Reply:
ok, but by using ` "phpunit/phpunit": "^7 | ^9",` we should be able to stay on 7.2 since phpunit 7 requires php 7.1

Reply:
done